dev-client/src/screens/SlopeScreen/SlopeScreen.tsx

```diff
@@ -54,26 +55,29 @@ export const SlopeScreen = ({siteId}: {siteId: string}) => {
   );
 
   return (
-    <Column space="1px">
-      <Row backgroundColor="primary.contrast" p="15px" alignItems="center">
-        <Heading variant="h6">{t('slope.title')}</Heading>
-        {/* TODO */}
-        <FormTooltip icon="info">Unimplemented tooltip</FormTooltip>
-      </Row>
-      <DataInputSummary
-        required={required}
-        complete={steepnessValue !== undefined}
-        label={t('slope.steepness.short_title').toLocaleUpperCase()}
-        value={steepnessValue}
-        onPress={onSteepness}
-      />
-      <DataInputSummary
-        required={required}
-        complete={shapeValue !== undefined}
-        label={t('slope.shape.title').toLocaleUpperCase()}
-        value={shapeValue}
-        onPress={onShape}
-      />
-    </Column>
+    <>
+      <Column space="1px">
+        <Row backgroundColor="primary.contrast" p="15px" alignItems="center">
+          <Heading variant="h6">{t('slope.title')}</Heading>
+          <InfoModal Header={t('slope.info.title')}>
+            <SlopeInfoContent />
+          </InfoModal>
+        </Row>
+        <DataInputSummary
+          required={required}
+          complete={steepnessValue !== undefined}
```

Comment:
question: can steepnessValue have other falsy values, or does TypeScript prevent that?

Reply:
Not sure; this diff is misleading -- I didn't add this code, just nested it in a no-longer-useful React fragment that I have since removed. So I'm inclined to not look into this for this task; let me know if you disagree.